posix-configs/bebop/px4.config

```diff
@@ -4,23 +4,27 @@ param load
 param set SYS_AUTOSTART 4013
 param set MAV_BROADCAST 1
 param set MAV_TYPE 3
+
 param set MPC_XY_VEL_P 0.12
 param set MPC_XY_P 1.3
-param set MPC_XY_VEL_D 0.002
+param set MPC_XY_VEL_D 0.006
 param set MPC_XY_VEL_I 0.05
-param set MC_ROLL_P 6.5
```

Comment:
There is no replacement for this - shouldn't be the value still set for ROLL / PITCH?